bids/tests/test_visualize.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+
+import shutil
+
+import pytest
+
+from bids.layout import BIDSLayout
+from bids.visualize import EventPlotter  # type: ignore
+from bids.visualize import LayoutPlotter  # type: ignore
+from bids.tests import get_test_data_path
+
+"""TODO
+-   test on datasets with a large range of events types
+"""
+
+
+@pytest.mark.parametrize(
+    "output_dir",
+    [
+        (Path(__file__).parent.joinpath("tmp")),
+        (None),
+    ],
+)
```

Comment:
Why is this needed? And why are we putting paths inside the source tree? The test failure is due to a race condition where multiple processes are creating the same "temporary" directory and deleting it.

The easiest way to get temporary paths in pytest is:

```Python
def test_name(tmp_path):
    # tmp_path is a Path() pointing to a unique temporary directory that will be deleted after the test is run
```